packages/components/fields/money-field/src/messages.js

```diff
@@ -1,9 +0,0 @@
-import { defineMessages } from 'react-intl';
-
-export default defineMessages({
-  highPrecision: {
```

Comment:
This is duplicated in the `money-input` component, and is not used in the money field.

```
warning [FormatJS CLI] Duplicate message id: "UIKit.MoneyInput.highPrecision", but the `description` and/or `defaultMessage` are different.
```